webapp/package.json

```diff
@@ -56,7 +56,6 @@
     "jest": "27.5.1",
     "jest-canvas-mock": "2.1.0",
     "jest-junit": "6.4.0",
-    "js-to-ts": "1.0.13",
     "react-js-to-ts": "1.4.0",
```

Comment:
It seems `react-js-to-ts` may also be unused?

Reply:
sure @mickmister Creating a PR for this as well.